libmachine/util/util.go

```diff
@@ -30,3 +31,32 @@ func GetProxyURL(hostUrl string) (*url.URL, error) {
 	}
 	return proxy, nil
 }
+
+// SplitArgs splits the given args slice into two slices. The first slice will contain all arguments before the special
+// "--" argument separator, and the second will contain all arguments after it.
+// For example, if the arguments are
+//
+//	"./machine host1 host2 -- --driver-flag driver-arg"
+//
+// then the first slice of arguments will be
+//
+//	"./machine host1 host2"
+//
+// and the second slice of arguments will be
+//
+//	"--driver-flag driver-arg"
+func SplitArgs(args []string) ([]string, []string) {
```

Comment:
if we return Args, which is basically []string,  we can use its methods like ` First()` and `Tail()` 
```suggestion
func SplitArgs(args []string) (Args, Args) {
```